### PR TITLE
Add Triangulation::n_global_coarse_cells()

### PR DESCRIPTION
**Pull request by peterrum**
... and use it in a couple of places

### CODE REVIEW

**Comment by drwells** [comment]
Could we move this to `NumberCache`?

**Comment by peterrum** [comment]
I was thinking about the same. But this was more easy to implement, since only p:f:T needs a special treatment. But I still can change it.

**Comment by drwells** [comment]
I would prefer for this to be in `NumberCache` since that is where we put everything else like this.

**Review by kronbichler — APPROVED** [review]
I agree with @drwells about putting it into the number cache, apart from that this looks good.

**Comment by peterrum** [comment]
@drwells @kronbichler Done!

File: source/distributed/tria_base.cc [hunk 1]
```diff
@@ -111,6 +111,7 @@ namespace parallel
   template <int dim, int spacedim>
   TriangulationBase<dim, spacedim>::NumberCache::NumberCache()
     : n_locally_owned_active_cells(0)
+    , number_of_global_coarse_cells(0)
```
Comment (drwells) [inline source/distributed/tria_base.cc:114]: I think you forgot to push something.

Comment (peterrum) [reply source/distributed/tria_base.cc:114]: Why? This is the NumberCache.

Comment (drwells) [reply source/distributed/tria_base.cc:114]: You're right - my mistake

**Review by drwells — APPROVED** [review]

**Pull request merged**